test/unit/io/stream/ostream_test_template.hpp

```diff
@@ -41,7 +42,10 @@ TYPED_TEST_P(ostream, output)
     std::string buffer{std::istreambuf_iterator<char>{fi}, std::istreambuf_iterator<char>{}};
 
     if constexpr (TestFixture::zero_out_os_byte)
+    {
         buffer[9] = '\x00'; // zero-out the OS byte.
+        SEQAN3_TEST_GTEST_SKIP_ZLIB_DEFLATE;
```

Comment:
The template is also used for `bz2`, which we don't want to skip.
`zero_out_os_byte` (also used for istream template) is currently equivalent to `is_zlib_compression`.